tests/unit/dataactvalidator/test_fabs38_detached_award_financial_assistance_3.py

```diff
@@ -33,18 +34,27 @@ def test_success(database):
     # Rule ignored for CorrectionDeleteIndicator of D
     det_award_5 = DetachedAwardFinancialAssistanceFactory(awarding_office_code=None, action_type='A',
                                                           action_date='10/01/2018', correction_delete_indicatr='D')
-    errors = number_of_errors(_FILE, database, models=[det_award_1, det_award_2, det_award_3, det_award_4, det_award_5])
+
+    # Rule ignored for action type is None
```

Comment:
Should we make sure it's being ignored for other action types as well as `None`?

Same as the other PR, some of the tests above are testing using a `None` action type

Reply:
Updated the Nones for the earlier tests.
Also `# Rule ignored for other action types` above.